client/js/stores/month-store.js

```diff
@@ -2,8 +2,33 @@ var Dispatcher = require('./dispatcher/dispatcher');
 var AppConstants = require('./constants/constants');
```

Comment:
I noticed you changed the name of the constants file, you will need to change the require statement on line 2.